packages/contracts/contracts/L1/rollup/OVM_CanonicalTransactionChain.sol

```diff
@@ -273,24 +275,31 @@ contract OVM_CanonicalTransactionChain is iOVM_CanonicalTransactionChain, Lib_Ad
             "Transaction gas limit too low to enqueue."
         );
 
-        // We need to consume some amount of L1 gas in order to rate limit transactions going into
-        // L2. However, L2 is cheaper than L1 so we only need to burn some small proportion of the
-        // provided L1 gas.
-        uint256 gasToConsume = _gasLimit/L2_GAS_DISCOUNT_DIVISOR;
-        uint256 startingGas = gasleft();
-
-        // Although this check is not necessary (burn below will run out of gas if not true), it
-        // gives the user an explicit reason as to why the enqueue attempt failed.
-        require(
-            startingGas > gasToConsume,
-            "Insufficient gas for L2 rate limiting burn."
-        );
+        // Transactions submitted to the queue are not routed through the Sequencer Entrypoint,
```

Comment:
There isn't a sequencer entrypoint anymore on the `experimental` branch, its 1:1 with L1 in that regard, so I'm not sure this comment makes sense to people who aren't familiar with the old system. Perhaps there is no need to mention that contract